icons/folder-gulp.svg

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The red of the folder itself is too dark and the color of the bottle should be a very light color (100) of the same red hue as the folder icon in the background.

<img src="https://user-images.githubusercontent.com/12248527/95854940-f2196b00-0d57-11eb-9fb7-11f9599b4ed3.png" width="500px" alt="material color palette"/>

Color palette can be found here:
https://pkief.github.io/material-color-converter/

Reply:
Thanks for the tool!